applications/Chat/evaluate/README.md

```diff
@@ -12,12 +12,13 @@ pip install -r requirements.txt
 
 ## Evaluation Pipeline
 
-The whole evaluation pipeline consists of two methods:
+The whole evaluation pipeline consists of three methods:
 
 1. `GPT Evaluation`: evaluates model predictions using GPT models.
    * Compare the performance of two different models (battle).
    * Rate the model according to pre-defined metrics using prompting design.
 2. `Automatic Evaluation`: evaluates model predictions using automatic metrics.
+3. `UniEval`: evaluates model predictions using UniEval models.
```

Comment:
Please add (English only) for this method.

Reply:
Ok.